assembl/static2/css/components/ideas.scss

```diff
@@ -0,0 +1,104 @@
+.ideas-level {
+    margin: 0 auto;
+    &.animating-towards-inline, &.is-inline {
+        position: relative;
+
+        // scroll containers are an empty area around the scroll button, so that user does not mis-click on idea link which is next to this button
+        .scroll-left-container, .scroll-right-container {
+            display: inline-block;
+            height: 80px;
+            width: 80px;
+            position: absolute;
+            top: calc(50% - 40px);
+            background-color: rgba(0,0,0,0);
+        }
+        .scroll-left-container {
+            left: 0;
+        }
+        .scroll-right-container {
+            right: 0;
+        }
+
+        .scroll-left, .scroll-right {
+            display: none;
+            position: absolute;
+            width: 50px;
+            height: 50px;
+            background-color: rgba(195, 195, 195, 0.6);
+            top: calc(50% - 25px);
+            cursor: pointer;
+            &:before {
+                font-size: 42px;
+                color: #000;
+                width: 100%;
+                text-align: center;
+                position: absolute;
+            }
+        }
+        .scroll-left {
+            z-index: 1000;
+            left: 0;
+            &:before {
+                content: "<";
+            }
+        }
+
+        &:hover {
+            .scroll-left, .scroll-right {
+                display: block;
+            }
+        }
+        .scroll-right {
+            z-index: 1001;
+            right: 0;
+            &:before {
+                content: ">";
+            }
+        }
+
+        .row {
+            position: relative;
+            display: inline-flex;
+            width: 100%;
+            overflow-x: hidden;
+
+            .theme {
+                max-width: 350px;
+                float: none;
+                display: inline-block;
+                flex-shrink: 0;
+                transition-duration: 1s;
+                left: 0;
+
+                &.clear {
+                    clear: none;
+                }
+
+                &:nth-child(1),
+                &:nth-child(2),
+                &:nth-child(3),
+                &:nth-child(4),
+                &:nth-child(5),
+                &:nth-child(6),
+                &:nth-child(7),
+                &:nth-child(8),
+                &:nth-child(9),
+                &:nth-child(10),
+                &:nth-child(11),
+                &:nth-child(12),
+                &:nth-child(13),
+                &:nth-child(14),
+                &:nth-child(15),
+                &:nth-child(16) {
+                    margin-top: 0;
+                    color: #fafbca;
+                }
+            }
+        }
+    }
+
+    .theme {
+        transition: all 0.5s;
```

Comment:
Yeah I think probability that someone asks to customize the transition durations of their debate is pretty low